proto/api.proto

```diff
@@ -514,6 +520,31 @@ message GetOrderbookResponse {
 message OrderbookItem {
   double price = 1;
   double size = 2;
+  string orderID = 3 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Serum generated OrderID"}];
+  uint64 clientOrderID = 4;
+  string ownerAddress = 5[(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "OpenOrders address for current market"}];
+}
+message GetMarketDepthRequest {
+  string market = 1;
+  uint32 limit = 2 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "Use 0 for no limits"}];
```

Comment:
I think we need a `depth` parameter? Are you using limit like that right now?

Reply:
if you mean the number of price points we return for asks/bids then yes - limit is used for that